iteration.go

```diff
@@ -231,3 +232,29 @@ func createIterationLinks(request *goa.RequestData, id interface{}) *app.Generic
 		Self: &selfURL,
 	}
 }
+
+// UpdateIterationsWithCounts accepts map of 'iterationID to a workitem.WICountsPerIteration instance'.
+// This function adds 'closed' and 'total' count of WI in relationship's meta for every given iteration in the slice.
+func UpdateIterationsWithCounts(data []*app.Iteration, wiCounts map[string]workitem.WICountsPerIteration) []*app.Iteration {
```

Comment:
@aslakknutsen yes earlier I tried to use `IterationConvertFunc` type. But that is defined as - `type IterationConvertFunc func(*goa.RequestData, *iteration.Iteration, *app.Iteration)`
And in my case I have different parameter list. If I change `IterationConvertFunc` to accept `interface{}` type and then I will be able to typecast it in my function as needed, but should I change that type ?